lisa/sut_orchestrator/azure/common.py

```diff
@@ -150,6 +150,8 @@
     "image_version",
 ]
 
+METADATA_ENDPOINT = "http://169.254.169.254/metadata/instance?api-version=2021-02-01"
```

Comment:
What's is this address for? How to make sure the IP works always?

Reply:
IMDS is a REST API that's available at a well-known, non-routable IP address (169.254.169.254).

Reply:
Please add code comments above the line to explain it.

Reply:
done